src/analysis/souffle/tests/arcs_fact_tests/diff_wrapper.sh

```diff
@@ -1,4 +1,21 @@
 #!/bin/bash
+#
+# Copyright 2015 The Bazel Authors. All rights reserved.
```

Comment:
I don't think this is right, this is one of our files right?

Reply:
Sorry, not sure where I got the idea that this was from Bazel. Thanks